CONTRIBUTING.md

```diff
@@ -64,6 +65,23 @@ Following these will ensure that your pull request is going to be accepted.
    This might seem pedantic but we believe that in essence this is going to improve overall comment and documentation quality.
 1. If possible try to sign your commits, e.g. using GPG keys. For more information about this go [here](https://help.github.com/en/articles/signing-commits).
 
+### Backwards Compatibility
+
+ink! and pallet_contracts are the projects under active development. As contracts API functions evolve to their newer versions, we need to introduce them in ink! in a way that still keeps current *MAJOR* ink! version compatible with elder substrate nodes built with the pallet_contracts version having not this new functionality.
```

Comment:
```suggestion
ink! and pallet_contracts are the projects under active development. As contracts API functions (denoted by the `seal_*` prefix) evolve to their newer versions, we need to introduce them in ink! in a way that still keeps current *MAJOR* ink! version compatible with older Substrate nodes. These older nodes may have a version of the Contracts pallet which does not support new seal APIs.
```

Reply:
left it to be Andrew's variant for the sake of concise